modules/storages/app/components/storages/admin/storage_list_component.sass

```diff
@@ -1,7 +1,7 @@
 .op-storage-list--header,
 .op-storage-list--row
   display: grid
-  grid-template-columns: 1fr 180px 100px 240px
+  grid-template-columns: 1fr 180px 160px 240px
```

Comment:
🟢  Perhaps we should even go for a full fractional measurement system 🤔

Reply:
The new width works 👍

Reply:
> 🟢 Perhaps we should even go for a full fractional measurement system 🤔

@akabiru  Do you know how the tables are supposed to react to smaller widths? Header and content are not aligned on smaller screens.

![storages-responsive-table](https://github.com/opf/openproject/assets/1113942/6c805b2a-1474-4908-acd4-897d9a5e63da)

Reply:
> > 🟢 Perhaps we should even go for a full fractional measurement system 🤔
> 
> @akabiru Do you know how the tables are supposed to react to smaller widths? Header and content are not aligned on smaller screens.
> 
> ![storages-responsive-table](https://user-images.githubusercontent.com/1113942/275724710-6c805b2a-1474-4908-acd4-897d9a5e63da.gif)

Good eye @dominic-braeunlein 👍🏾 we're solving part of the resize issues in https://community.openproject.org/wp/50307 I believe @Kharonus also has some ideas on the header and content mis-alignment. The discussions on the "primer like" table are WIP as primer doesn't offer a _real_ table yet.

The topic could use your insight for sure 😎

Reply:
@akabiru already answered it, the sizing is fixed in a different wp. 

regarding fractional column width: I didn't want to change current behavior too much, as I was not part of the latest discussion about the general behavior of that grid.